src/fiber/graph.rs

```diff
@@ -988,41 +949,6 @@ mod tests {
         }
     }
 
-    #[test]
-    fn test_graph_connected_peers() {
-        let temp_path = tempfile::tempdir().unwrap();
-        let store = Store::new(temp_path.path());
-        let keys = generate_keys(1);
-        let public_key1 = keys[0];
-        let mut network_graph = NetworkGraph::new(store, public_key1.into());
-
-        let peer_id = PeerId::random();
-        let address: Multiaddr = "/ip4/127.0.0.1/tcp/10000".parse().unwrap();
-        network_graph.add_connected_peer(&peer_id, address.clone());
-
-        let connected_peers = network_graph.get_connected_peers();
-        assert_eq!(connected_peers.len(), 1);
-        assert_eq!(connected_peers[0], (&peer_id, &address));
-
-        network_graph.reset();
-        let connected_peers = network_graph.get_connected_peers();
-        assert_eq!(connected_peers.len(), 0);
-
-        // load from db
-        network_graph.load_from_store();
-        let connected_peers = network_graph.get_connected_peers();
-        assert_eq!(connected_peers.len(), 1);
-        assert_eq!(connected_peers[0], (&peer_id, &address));
-
-        network_graph.remove_connected_peer(&peer_id);
-        let connected_peers = network_graph.get_connected_peers();
-        assert_eq!(connected_peers.len(), 0);
-
-        network_graph.load_from_store();
-        let connected_peers = network_graph.get_connected_peers();
```

Comment:
can we change this unit test for `PersistentNetworkActorState`.
just to make sure the apis work as expected will be good.

Reply:
I wanted to make a unit test for this. The problem now for me is that `NetworkActorState` is too complex. In order to create a `NetworkActorState`, we almost certainly need to create a `NetworkActor`.  The pr https://github.com/nervosnetwork/fiber/pull/205 actually has an easier way to create and restart a network actor. I think that would make the test easier. Admittedly, that is kind of like an integration test. And we still need to solve the problem of how to easily write unit tests for `NetworkActorState`.